src/operations.jl

```diff
@@ -198,11 +198,11 @@ function solve(o::Calibrate; callback)
             fit = EasyModelAnalysis.datafit(prob, init_params, o.data)
         else
             init_params = Pair.(EasyModelAnalysis.ModelingToolkit.Num.(first.(o.priors)), tuple.(minimum.(last.(o.priors)), maximum.(last.(o.priors))))
-            fit = global_datafit(prob, init_params, o.data)
+            fit = EasyModelAnalysis.global_datafit(prob, init_params, o.data)
         end
 
-        newprob = remake(prob, p=fit)
-        sol = solve(newprob)
+        newprob = EasyModelAnalysis.DifferentialEquations.remake(prob, p=fit)
+        sol = EasyModelAnalysis.DifferentialEquations.solve(newprob)
```

Comment:
```suggestion
        sol = EasyModelAnalysis.DifferentialEquations.solve(newprob; saveat = 1)
```

Let's give 1-spaced data out of global calibrate to in order to make things simpler?